src/app/components/legacy/FormatCredit/index.js

```diff
@@ -0,0 +1,23 @@
+import html from 'nanohtml';
+import { track } from '../../../utils/behaviour';
+import styles from './index.lazy.scss';
+
+const LINK_URL = '/news/interactives/';
+const LINK_TRACKER = () => track('format-credit-link', '*');
+
+const FormatCredit = () => {
+  styles.use();
+
+  return html`
+    <div class="FormatCredit--legacy">
+      <p>
+        <span class="FormatCredit__wrapper">
```

Comment:
Needed to add a wrapper for the same reason, to revert to original styles without specificity issues.